clang/test/CIR/CodeGen/struct-init.cpp

```diff
@@ -5,6 +5,21 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -emit-llvm %s -o %t.ll
 // RUN: FileCheck --check-prefix=OGCG --input-file=%t.ll %s
 
+struct O {
```

Comment:
Would love for this to have a better name :)  Just a few characters even.  Otherwise `O overlapping_init` looks a LOT like 1 word/was jarring.

Reply:
Yeah, I didn't like that either and I'm the one who wrote it! I'm not sure why I left it that way.